src/index.njk

```diff
@@ -31,12 +31,23 @@ sitemapPriority: 1.0
 </div>
 
 <!-- Secondary Content -->
-<div class="w-full ff-bg-light px-6 pt-12 pb-48 sm:pt-32 sm:px-0">
+<div class="w-full ff-bg-light px-6 pt-12 pb-48 sm:pt-32">
     <div class="container m-auto text-center max-w-4xl">
         <div class="m-auto mt-36 mb-12 sm:mt-64 sm:mb-16 w-full text-center">
             <h2>Why use FlowForge for Node-RED</h2>
             <h5><a href="/node-red">What is Node-RED?</a></h5>
         </div>
+        <div class="m-auto mb-12 sm:mb-16 w-full text-center flex flex-col md:flex-row">
+            <p class="md:w-1/6 md:text-left opacity-75">Trusted by:</p>
```

Comment:
Removing this element on the page I think is cleaner. WDYT?

Or should it link to the customer stories page?

Reply:
Yndira: I think we need to explain what these logos are doing here on this page.